### ISSUE TITLE
frequency database

### ISSUE THREAD

**Issue by jimmyoic** [chewing/chewing-editor#161]
I think maybe we can record the frequency of the words for choosing the vocabulary which the user may prefer when there is the same pronouncement in many vocabularies

For example in ㄇㄧˋㄐㄧˊ  we may have 密集 and 秘笈,  but the user may prefer the latter one since it is more often to be typed

**Comment by david50407** [comment]
Yes we have recorded the frequency in libchewing's database (ref: https://github.com/chewing/libchewing/blob/a495d845c82186c4ba7773ee8cc8bf4c94ac790e/src/userphrase-sql.c#L252).
But I don't know if we can change the frequency in editor or not.
@czchen is it possible in chewing?

**Comment by czchen** [comment]
@david50407 
It is possible, but user might not know how to set the frequency.

**Comment by kidwm** [comment]
@czchen libchewing should offer API to adjust weight of phrase higher than another, then we could let use arrange order by themselves.
